testing/camunda-process-test-java/src/test/java/io/camunda/process/test/api/UserTaskAssertTest.java

```diff
@@ -571,7 +571,7 @@ public void hasAssigneeFailureMessage() {
 
   @Nested
   public class DueDate {
-    private static final String DUE_DATE = "2025-05-01T10:00:00.000Z";
+    private static final String DUE_DATE = "2025-05-01T10:00:00.100Z";
```

Comment:
needed top ensure the toString is not omiting seconds and millis as they are all zero (default toString behavior on OffsetDateTime)